packages/fhir-group-management/src/components/CommodityList/index.tsx

```diff
@@ -123,3 +163,34 @@ export const CommodityList = (props: GroupListProps) => {
 
   return <BaseListView {...baseListViewProps} />;
 };
+
+/**
+ * Soft deletes a commodity resource. Sets its active to false and removes it from the
+ * list resource.
+ *
+ * @param fhirBaseURL - base url to fhir server
+ * @param obj - commodity resource to be disabled
+ * @param listId - id of list resource where this was referenced.
+ */
+export const deleteCommodity = async (fhirBaseURL: string, obj: IGroup, listId: string) => {
+  if (!listId) {
+    throw new Error('List id is not configured correctly');
+  }
+  const disabledGroup: IGroup = {
+    ...obj,
+    active: false,
+  };
+  const serve = new FHIRServiceClass<IGroup>(fhirBaseURL, groupResourceType);
+  const listServer = new FHIRServiceClass<IList>(fhirBaseURL, listResourceType);
+  const list = await listServer.read(listId);
+  const leftEntries = (list.entry ?? []).filter((entry) => {
```

Comment:
Is there a possibility to filter this list at display for values that have `active:false` rather than removing it from the payload every-time? The same way we do to practitioners

Reply:
@machariamuguku see this: 
> Remove the Group from the Commodity List resource.
from the issue description.

So I think the list resource is being used in configuring things for the mobile client. If we do not remove the commodity then that means that the client might be forced to fetch more resources than It needs, also that these commodity resources that it needs to fetch might grow unchecked if depening on how quickly they change.

Though @dubdabasoduba can clarify if there should be a change here.

Reply:
Yes, that's true. The list resource is used to tell the app what Group resources to download.  We need to remove the Groups from the list resource so we don't show the commodities that have already been removed